.env.sample

```
@@ -1,14 +1,16 @@
 # General URLs
-KANEO_CLIENT_URL=
-KANEO_API_URL=
+KANEO_CLIENT_URL=                          # e.g. http://localhost:5173
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)